api/v1/gostream.proto

```diff
@@ -68,7 +68,8 @@ message CreateUserResponse {
 }
 
 message UpdateUserRequest {
-  User user = 1;
+  uint64 user_id = 1;
+  User user = 2;
```

Comment:
Following the `UpdatePetRequest` format, this message should also be like 

```go
message UpdateUserRequest {
  uint64 user_id = 1;
  User new_data = 2;
  }
```

You can also add http options to the update service like:
```proto
option (google.api.http) = {
      put: "/user/{user_id}"
      body: "*"
    };
```